lib/instrumentation/span.js

```diff
@@ -27,19 +23,6 @@ function Span (transaction) {
   this.context = current.context.child()
 
   this._agent.logger.debug('init span %o', { id: this.transaction.id })
```

Comment:
This can be removed and we can keep the log line below from the old `start` function

Reply:
fixed